src/App.jsx

```diff
@@ -1,13 +1,180 @@
-
 import './App.css'
+import {useState} from "react";
 
 function App() {
+    //aarbeien
+    const [fruitCounter, setFruitCounter] = useState(0);
+    const increment = () => setFruitCounter((prev) => prev + 1);
+    const decrement = () => setFruitCounter((prev) => Math.max(0, prev - 1));
+    const eventHandler = (e) => {
+        const value = Math.max(0, e.target.value);
+        setFruitCounter(value);
+    }
+    //bananen
+    const [fruitCounterb, setFruitCounterb] = useState(0);
+    const incrementb = () => setFruitCounterb((prev) => prev + 1);
+    const decrementb = () => setFruitCounterb((prev) => Math.max(0, prev - 1));
+    const eventHandlerb = (e) => {
+        const value = Math.max(0, e.target.value);
+        setFruitCounterb(value);
+    }
+    //appels
+    const [fruitCounterc, setFruitCounterc] = useState(0);
+    const incrementc = () => setFruitCounterc((prev) => prev + 1);
+    const decrementc = () => setFruitCounterc((prev) => Math.max(0, prev - 1));
+    const eventHandlerc = (e) => {
+        const value = Math.max(0, e.target.value);
+        setFruitCounterc(value);
+    }
+    //kiwi's
+    const [fruitCounterd, setFruitCounterd] = useState(0);
+    const incrementd = () => setFruitCounterd((prev) => prev + 1);
+    const decrementd = () => setFruitCounterd((prev) => Math.max(0, prev - 1));
+    const eventHandlerd = (e) => {
+        const value = Math.max(0, e.target.value);
+        setFruitCounterd(value);
+    }
+    const resetCounter = () => {
+        setFruitCounter(0);
+        setFruitCounterb(0);
+        setFruitCounterc(0);
+        setFruitCounterd(0);
+    };
+    // state van de formulieer gegevens
+    const [voorNaam, setVoorNaam] = useState("");
+    const [achterNaam, setAchterNaam] = useState("");
+    const [leeftijd, setLeeftijd] = useState(0);
+    const [postcode, setPostcode] = useState("");
+    const [bezorgfrequentie, setBezorgFrequentie] = useState("");
+    const [tijdvak, setTijdvak] = useState("");
+    const [opmeringen, setOpmerkingen] = useState("");
+    const [akkoord, setAkkoord] = useState(false);
+
+
+    //bijwerken van de formuliergegevens
+    const formHandlerVoorNaam = (e) => {
+        setVoorNaam(e.target.value);
+    };
+    const formHandlerAchterNaam = (e) => {
+        setAchterNaam(e.target.value);
+    };
+    const formHandlerLeeftijd = (e) => {
+        setLeeftijd(e.target.value);
+    };
+    const formHandlerPostcode = (e) => {
+        setPostcode(e.target.value);
+    };
+    const formHandlerBezorgFrequentie = (e) => {
+        setBezorgFrequentie(e.target.value);
+    };
+    const formHandlerTijdvak = (e) => {
+        setTijdvak(e.target.value);
+    };
+    const formHandlerOpmerkingen = (e) => {
+        setOpmerkingen(e.target.value);
+    };
+    const formHandlerAkkoord = (e) =>
+    {
+        setAkkoord(e.target.checked);};
+
```

Comment:
De logica is hier ook goed, maar ook dit kan efficiënter. Je zou genoeg moeten hebben aan 1 enkele functie. Het is zelfs niet eens noodzakelijk om een functie te gebruiken. Je kan de setter van de state ook direct in de onChange methode aanroepen.

vb: 

`onChange={(e) => setVoorNaam(e.target.value)`